draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -451,6 +453,12 @@ ExternalMu-ML-DSA.Sign_internal(sk, mu, rnd):
 ~~~
 {: #fig-externalmu-ml-dsa-internal title="Internal steps of ExternalMu-ML-DSA"}
 
+There are some implementation considerations and security specific to the ExternalMu mode that are worth discussion.
+
+An additional implementation consideration imposed by ExternalMu-ML-DSA is the requirement to provide the public key, or its prehash, to the `ExternalMu-DL-DSA.Prehash()` routine. This is different from conventional pre-hashing which only requires the message and the hash function as input. This may impose additional implementation burden in order for the client that is performing the pre-hash to first obtain the public key prior to invoking the pre-hash routine. Implementions are RECOMMENDED to expose interfaces for both ML-DSA and ExternalMu-ML-DSA so that applications can manage themselves the tradeoff between ExternalMu pre-hashing when they have the public key available, and pure ML-DSA which requires streaming the entire message into the cryptographic module.
```

Comment:
ExternalMu-DL-DSA.Prehash()  - typo

- language typos... Check over.

Reply:
I suggest to remove the normative language here. Good to point out that the entity that does the prehashing and requests for the signature has the public key beforehand 

> ExternalMu-ML-DSA requires the public key, or its prehash, as input to the pre-digesting function. This assumes the signer generating the pre-hash is in possession of the public key before signing and is different from conventional pre-hashing which only requires the message and the hash function as input and .

Reply:
@johngray-dev Did you confirm there was a typo?

Reply:
I think this is getting resolved via Panos' re-write.